modules/portmaster/src/i18n/helptexts.yaml

```diff
@@ -41,95 +40,92 @@ uiMode:
 pilot-widget-NetworkRating:
   title: Network Rating
   content: |
-    Some features and settings of the Portmaster behave differently, depending on the environment.
+    Control your privacy even when connecting to new networks.
 
-    In order to help the Portmaster provide the most adequate protection, please select "Trusted", "Untrusted" or "Danger" according to the situation you are in.<br>
-    Continue to learn about these different levels.
+    In the Portmaster you configure settings to be active in one environment but not in the other, like allowing sensitive connections at home but not at the public library.
+
+    The only thing you have to do is to change the network rating whenever you connect to a different network.
```

Comment:
As mentioned earlier, this may change while being connected.

Reply:
same point applies as mentioned in the other discussion